File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include<stdio.h>
int main(){
	int N;
	scanf("%d",&N);
	int A[N];
	int min,max;
	
	for(int i=0;i<N;i++){
		scanf("%d",&A[i]);
		if(i==0) min=A[i],max=A[i];
	}
	for(int i=0;i<N;i++){
		if(A[i]<min) min=A[i];
		if(A[i]>max) max=A[i];
	}
	printf("%d %d",min,max);
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
import sys

n = int(sys.stdin.readline())
a = list(map(int, sys.stdin.readline().split()))
min, max = int(a[0]), int(a[0])

for i in a:
    if min > int(i):
        min = i
    elif max < int(i):
        max = i

print(min, max)